lib/adzerk/channel.rb

```diff
@@ -0,0 +1,9 @@
+module Adzerk
+    class Channel < ApiEndpoint
+        def get_priorities(channel_id)
+            url = "channel/#{channel_id}/priorities"
+            response = client.get_request(url)
+            JSON.parse(response.body)
```

Comment:
If possible, this should be using `parse_response` rather than directly calling `JSON.parse`.

Reply:
Updated